### ISSUE TITLE
Launching .tar does nothing

### ISSUE THREAD

**Issue by ieatfrisbees** [Wolfsblvt/BlossomsPokemonGoManager#110]
Using Windows 10 with the proper, latest version of Java clean-installed. Have restarted my computer a few times and made sure there's no lingering files left from old version of Java. Yet when I ask Windows to open the .tar archive, a window flashes briefly on-screen and disappears (can't tell what it is) and then nothing else happens.

Any ideas?

**Comment by Lacero** [comment]
At least in Windows 7, .tar.gz files aren't supported by default. It's really a Unix file format. Go with the .zip source or install a compression utility such as 7-Zip.

**Comment by ieatfrisbees** [comment]
I'm using Winrar to unzip the downloaded file. Maybe I'm trying to run the wrong file in Java? What file specifically should I be looking to run?

**Comment by Wolfsblvt** [comment]
Inside the zip, there is a file that's called something like BlossomsPokemonGoManager.jar, that file has to be called.

**Comment by ieatfrisbees** [comment]
Alright, well, I was originally trying to use the command prompt to launch everything, but that wouldn't work and I kept getting errors. So that's why I've been trying to use the method that was mentioned on the main page here for Windows 10 which was right-clicking on the file and trying to open with Java, but that wasn't working, as I explained here. I just tried using the command prompt to launch it again and it works fine now (after deleting the config.json file). No idea why it works now but didn't before. 

Oh well. Thanks for your help and quick replies, all!! Appreciate it.
